sewer/client.py

```diff
@@ -722,10 +713,8 @@ def get_certificate(self):
                 identifier_auth = self.get_identifier_authorization(url)
                 token = identifier_auth["token"]
                 acme_keyauthorization = self.get_keyauthorization(token)
-                cleanup_kwargs = self.auth_provider.fulfill_authorization(
-                    identifier_auth, token, acme_keyauthorization
-                )
-                cleanup_kwargs_list.append(cleanup_kwargs)
+                self.auth_provider.setup_auth(identifier_auth, token, acme_keyauthorization)
```

Comment:
I'd like to see how `setup_cert` would fit in here, since its counterpart `clear_cert` is being used below